proto/server/configEvtHandler.go

```diff
@@ -191,6 +194,15 @@ func handleNetworkSlicePost(configMsg *configmodels.ConfigMessage, subsUpdateCha
 	rwLock.Unlock()
 }
 
+func firstConfigReceived() bool {
+	if len(getDeviceGroups()) > 0 {
+		return true
+	} else if len(getSlices()) > 0 {
+		return true
+	}
+	return false
```

Comment:
What about this change?
```suggestion
	return len(getDeviceGroups()) > 0 || len(getSlices()) > 0
```

Reply:
Applied the suggestion, thank you!

Reply:
It seems that this change got reverted. Or is it intentional?